src/notation/view/abstractnotationpaintview.cpp

```diff
@@ -81,6 +81,9 @@ AbstractNotationPaintView::~AbstractNotationPaintView()
         m_notation->accessibility()->setMapToScreenFunc(nullptr);
         m_notation->interaction()->setGetViewRectFunc(nullptr);
     }
+
+    m_previousSelectedElement = nullptr;
+    delete m_previousSelectedElement;
```

Comment:
This `delete` looks like it should not be there! (because 1. it deletes a nullptr and 2. we should not delete elements because that's handled by the Engraving module and the UndoStack)